Recipe/Recipe.Core/Helper/PlatformHelper.cs

```diff
@@ -0,0 +1,84 @@
+﻿using System.Web;
+
+namespace Recipe.Core.Helper
+{
+    public static class PlatformHelper
+    {
+        public static string GetUserPlatform(HttpRequest request)
+        {
+            var ua = request.UserAgent;
+
+            if (ua.Contains("Android"))
```

Comment:
Can we create a Dictionary or a List for possible OS Versions. Instead of having condition for every version?

Reply:
I believe that would be a performance overhead, also we don't have complete key which needs to be matched in user agent so by putting dictionary into it we would need to iterate every single element from dictionary.

Reply:
I didn't emphasize on the dictionary. That's why I said Dictionary or list. My point was to better manage this code. You can also go with a single regex for all and fetch the group from it.

Just for my understanding: I would love to hear about the performance overhead with the dictionary.